addon/mixins/shared-stylist.js

```diff
@@ -1,79 +1,79 @@
 import Ember from 'ember';
-const { keys } = Object;
-const {observer, A, run, typeOf, on} = Ember;
-const _styleProperties = ['maxWidth', 'width', 'minWidth','height','fontSize','fontFamily','fontWeight','fontStyle','cursor'];
-const GOLDEN_RATIO = 1.618;
-const ASPECT_RATIO = 1.3;
-const sizer = size => {
-  return Number(size) === size ? size + 'px' : size;
-};
-/**
- * Ensures that the value has been escaped before allowing through to the DOM
- * @param  {mixed}  input API or addon provided value
- * @return {string}       HTML escaped string
- */
-const securitize = function(input){
-  if(typeOf(input) === 'object') {
-    input = input.toString();
-  }
-  return Ember.Handlebars.Utils.escapeExpression(input);
-};
 
-var SharedStylist = Ember.Mixin.create({
-  _styleWhitelist: _styleProperties,
-  _unbindStyle: on('didInitAttrs',function() {
-    new A(this.get('attributeBindings')).removeObject('style');
+export default Ember.Mixin.create({
```

Comment:
I see you've moved away from a named mixin? Why? The approach I've taken will expose the name of the mixin to the Ember Inspector rather than the default "unknown mixin"